Frends.ManagementApi.Request/Frends.ManagementApi.Request/Frends.ManagementApi.Request.csproj

```diff
@@ -0,0 +1,45 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+	  <TargetFramework>net6.0</TargetFramework>
+	  <LangVersion>Latest</LangVersion>
+	  <Version>1.0.0</Version>
+	  <Authors>Frends</Authors>
+	  <Copyright>Frends</Copyright>
+	  <Company>Frends</Company>
+	  <Product>Frends</Product>
+	  <PackageTags>Frends</PackageTags>
+	  <PackageLicenseExpression>MIT</PackageLicenseExpression>
+	  <GenerateDocumentationFile>true</GenerateDocumentationFile>
+	  <Description>Task for reading data from Frends Management API.</Description>
```

Comment:
I guess this task should do a bit more than just reading data, unless there will be separate Tasks for POST, PUT and PATCH methods.